test/kpod_ps.bats

```diff
@@ -3,6 +3,10 @@
 load helpers
 
 IMAGE="redis:alpine"
+function setup() {
+    copy_images
```

Comment:
We only use the `redis:alpine` image in this file, so I don't think this benefits it.